src/utilities/embeddable-utils.ts

```diff
@@ -74,6 +74,9 @@ export const getAnswerWithMetadata = (
     // LARA also seems to be storing the interactiveState as a JSON string and that
     // JSON string is what is used in the reportState
     interactiveState: JSON.stringify(interactiveState),
+    interactive: {
+      id: embeddable.ref_id
```

Comment:
In the other place in the state the id is included it uses `refIdToAnswersQuestionId` to convert the id if necessary.
Can you look into this to understand what format the id needs to be in for things to work in the report?

If this id needs to be a in the `123-MangedInteractive` format instead of `managed_interactive_123`, then please update the interactive ids document. It currently refers to `adding an answer to the Report Service` but doesn't specify that the id actually appears in 2 places in the answers. https://docs.google.com/document/d/1vhRfcrxPRMtJRkHDqYbDPXKPDCqZV5JjjKoSmqB5zMg/edit